### PR TITLE
Optimize various SVGs (second try)

### PR DESCRIPTION
**Pull request by RiedleroD**
So last time didn't work - for more context see #2130 and #2179.

I only fixed the circlecount image for now, to see if this hypothetical fix even works. If this doesn't work then your SVG engine is severely outdated or just plain shit.

Anyway, @borgmanJeremy could you please check if the circlecount image displays correctly now? Compat. commit is 038f869a5bb2cfbccd9a7f06c74b269233e7b2be

### CODE REVIEW

**Comment by borgmanJeremy** [comment]
I've only tested on MacOs but the circle counter looks correct now.

**Comment by RiedleroD** [comment]
that should be all, I think. Everything should work now.

**Comment by borgmanJeremy** [comment]
Can you squash and rebase on master? As an aside please don't have obscenities in the final commit message, I try to keep it family friendly 😁

**Comment by RiedleroD** [comment]
> Can you squash and rebase on master

wish me luck :/ git usually doesn't like me

> please don't have obscenities in the final commit message

got it! Wouldn't have added any, since that one is actually important, but in the future I'll try to keep them out of the regular ones as well 👍🏻

**Comment by borgmanJeremy** [comment]
If git gives you a problem let me know I can fix it

**Comment by RiedleroD** [comment]
ayy it worked!

**Comment by borgmanJeremy** [comment]
Thanks! Just tested this on MacOS and its 99% correct. The only issue is with the "upload to imgur icon". The massive keyboard looked good too. 


I'll test on Windows tomorrow.

**Comment by RiedleroD** [comment]
should be fixed now, will test on windows later today as well

**Comment by RiedleroD** [comment]
tested x64 portable on windows, works without visible problems.

**Comment by borgmanJeremy** [comment]
All looks good here. Thanks!!

**Review by borgmanJeremy — APPROVED** [review]

**Comment by borgmanJeremy** [comment]
Do you have directions for how you generated these reduced SVGs? If i make more icons in the future I can minimize them.

**Comment by RiedleroD** [comment]
> generated

If you could see the bags under my eyes you wouldn't ask that question.

On a more serious note, most of it is just knowing what stuff you can remove without problems, various tricks to make paths shorter and a few handy websites: [yqqn's path editor](https://yqnn.github.io/svg-path-editor/), [ellipse/circle to path converter](http://complexdan.com/svg-circleellipse-to-path-converter/), and inkscape if there's some path transform you can't do otherwise. There's also various websites and applets that try to minimize SVGs automagically, but I haven't had much success with them in the past.

TL;DR: it's mostly experience.

**Comment by borgmanJeremy** [comment]
Haha that shows my ignorance for graphic design. I usually draw it in GIMP and then auto trace it in Inkscape....

**Comment by RiedleroD** [comment]
you can send me a message when you need a new icon → riedler@riedler.wien
